### PR TITLE
Feature/18 create apps header

### LINKED ISSUE #18: Feature: Create app's header
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

image

image

**Describe alternatives you've considered**

**Additional context**
Web Frontend
Version is loaded dynamically.

### PR DESCRIPTION
**Pull request by Lukash88**
## Summary

App's header component has been created and added.

## Details

App's header has been added to the application's frontend according to page layout mockup.
Version of app is mocked up.

image

## References

No external references needed.

### CODE REVIEW

**Comment by StraykerPL** [comment]
Disconnect PR from backlog.

File: binder-web-frontend/src/pages/home/home.component.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
-<div>
-  <navbar></navbar>
-  <tables-list></tables-list>
-  <table-view></table-view>
-  <domain-buttons></domain-buttons>
-</div>
+<div class="container">
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/home.component.html]: Out of scope work performed. Please stick to ticket's request.

File: binder-web-frontend/src/pages/home/home.component.scss
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/home.component.scss:1]: This is Web Framework - please use Sass in SCSS notation.

Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/home.component.scss:1]: Out of scope work performed. Please stick to ticket's request.

File: binder-web-frontend/src/shared/components/app-header/app-header.component.html
Comment (StraykerPL) [inline binder-web-frontend/src/shared/components/app-header/app-header.component.html:1]: Move component to Home page.

File: binder-web-frontend/src/shared/components/app-header/app-header.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div>
+    <h2> Binder <span>v{{version}}</span></h2>
```
Comment (StraykerPL) [inline binder-web-frontend/src/shared/components/app-header/app-header.component.html]: Remove first space, add spaces between version brackets.

File: binder-web-frontend/src/shared/components/app-header/app-header.component.scss
Comment (StraykerPL) [inline binder-web-frontend/src/shared/components/app-header/app-header.component.scss:1]: This is Web Framework - please use Sass in SCSS notation.

File: binder-web-frontend/src/shared/components/app-header/app-header.component.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+import  packageInfo  from 'package.json';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './app-header.component.html',
+  styleUrls: ['./app-header.component.scss']
+})
+export class AppHeaderComponent {
+
```
Comment (StraykerPL) [inline binder-web-frontend/src/shared/components/app-header/app-header.component.ts]: Remove empty line.

File: binder-web-frontend/tsconfig.json
Comment (StraykerPL) [inline binder-web-frontend/tsconfig.json:1]: Do not change framework's config.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-frontend/src/shared/components/app-header/app-header.component.html (hunk 2 above)
Comment (Lukash88) [reply binder-web-frontend/src/shared/components/app-header/app-header.component.html]: Resolved 1097a94

File: binder-web-frontend/src/shared/components/app-header/app-header.component.ts (hunk 3 above)
Comment (Lukash88) [reply binder-web-frontend/src/shared/components/app-header/app-header.component.ts]: Resolved cecbfb5

File: binder-web-frontend/src/shared/components/app-header/app-header.component.html
Comment (Lukash88) [reply binder-web-frontend/src/shared/components/app-header/app-header.component.html:1]: Resolved 230ec33

File: binder-web-frontend/src/pages/home/home.component.html (hunk 1 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/home.component.html]: Resolved 2f3505b

File: binder-web-frontend/src/pages/home/home.component.scss
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/home.component.scss:1]: Resolved ce6188d

Comment (Lukash88) [reply binder-web-frontend/src/pages/home/home.component.scss:1]: Resolved ce6188d

File: binder-web-frontend/src/shared/components/app-header/app-header.component.scss
Comment (Lukash88) [reply binder-web-frontend/src/shared/components/app-header/app-header.component.scss:1]: Resolved bfa3eee

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<div>
+    <h2>Binder <span>v{{ version }}</span></h2>
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/app-header/app-header.component.html]: Switch styling from tag to style sheet.

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+div {
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/app-header/app-header.component.scss]: Do not use tags, replace with BEM classes.

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+import  packageInfo  from 'package.json';
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts]: Remove.

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './app-header.component.html',
+  styleUrls: ['./app-header.component.scss']
+})
+export class AppHeaderComponent { 
+   version: any = packageInfo.version;
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts]: Make version of type number, mock value.

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+import  packageInfo  from 'package.json';
+
+@Component({
+  selector: 'app-header',
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts]: Remove prefix from component.

File: binder-web-frontend/src/pages/pages.module.ts
Comment (StraykerPL) [inline binder-web-frontend/src/pages/pages.module.ts:1]: If component is in Home page component, declare it in page's module, not the shared one.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-frontend/tsconfig.json
Comment (Lukash88) [reply binder-web-frontend/tsconfig.json:1]: Resolved b988762

File: binder-web-frontend/src/pages/pages.module.ts
Comment (Lukash88) [reply binder-web-frontend/src/pages/pages.module.ts:1]: Resolved 8d54cc3

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts (hunk 8 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts]: Resolved 8d54cc3

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts (hunk 7 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts]: Resolved 8d54cc3

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts (hunk 6 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/app-header/app-header.component.ts]: Resolved 8d54cc3

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.scss (hunk 5 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/app-header/app-header.component.scss]: Resolved 8d54cc3

File: binder-web-frontend/src/pages/home/components/app-header/app-header.component.html (hunk 4 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/app-header/app-header.component.html]: Resolved 8d54cc3

File: binder-web-frontend/src/pages/home/components/header/header.component.scss [hunk 9]
```diff
@@ -0,0 +1,10 @@
+.header {
+  &__title {
+    font-size: 1.5rem;
+    border-bottom: solid 0.2rem #fff;
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/header/header.component.scss]: Change color code to default, 6-char length.

File: binder-web-frontend/src/shared/components/.gitkeep
Comment (StraykerPL) [inline binder-web-frontend/src/shared/components/.gitkeep:1]: If folder stays empty, do not delete this file.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-frontend/src/pages/home/components/header/header.component.scss (hunk 9 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/header/header.component.scss]: Resolved 483cd16

File: binder-web-frontend/src/shared/components/.gitkeep
Comment (Lukash88) [reply binder-web-frontend/src/shared/components/.gitkeep:1]: Resolved 48714c9

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**